controls/sysctl_spec.rb

```diff
@@ -311,6 +314,9 @@
   describe kernel_parameter('net.ipv6.conf.default.autoconf') do
     its(:value) { should eq 0 }
   end
+  describe kernel_parameter('net.ipv6.all.default.autoconf') do
```

Comment:
Should this be `net.ipv6.conf.all.autoconf`?

https://docs.vmware.com/en/vRealize-Operations/8.6/com.vmware.vcom.core.doc/GUID-37B91C4A-5E1E-4F8E-BC59-B3552BA7CDFA.html

Reply:
Thanks @raman-gill-acl!